packages/bpk-component-navigation-bar/src/BpkNavigationBar.tsx

```diff
@@ -50,6 +54,8 @@ const BpkNavigationBar = (props: Props) => {
     leadingButton,
     sticky = false,
     title,
+    titleTagName,
+    titleTextStyle,
```

Comment:
As you have made these optional, we will need to set these up as defaults otherwise it is going to cause issues when not provided. 

That way you don't need to have what you have below and can just pass the props through.

Currently though there isn't even a fallback for `titleTagName` so that might cause some havoc

I would say here it probably should be

```suggestion
    titleTagName = 'span',
    titleTextStyle = TEXT_STYLES.heading5,
```